docs/release/release_0_4_9.md

```diff
@@ -0,0 +1,105 @@
+# napari 0.4.9
+
+We're happy to announce the release of napari 0.4.9!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+
+## Highlights
+This release adds a couple nice new features like additional shading modes for our
+surface layer (#2972) and the ability to copy a screenshot directly to the clipboard (#2721).
+It also contains a variety of bug fixes and improvements.
+
+
+## New Features
+- Added copy-to-clipboard functionality (#2721)
+- Add `make watch` command for hot reload (#2763)
+- Expose alternative shading modes for surfaces (#2792)
+
+
+## Improvements
+- Global plugin setting (#2565)
+- Delay all imports in `napari.__init__` behind module level `napari.__getattr__` (#2662)
+- Add `block` to viewer.show (#2669)
+- New type stubs PR, and simpler `napari.view_layers` module (#2675)
+- Add `MultiScaleData` wrapper to give multiscale data a consistent API (#2683)
+- Revert "add `MultiScaleData` wrapper to give multiscale data a consistent API (#2683)" (#2807)
+- Add repr-html to nbscreenshot (#2740)
+- Set default highlight thickness to 2 (#2746)
+- Save shortcuts in settings (#2754)
+- Enable correct loading of settings from environment variables (#2759)
+- Add tooltips to widgets in preferences (#2762)
+- Improve colormap error message, when using display names or wrong colormap names (#2769)
+- Add parent to console and dockwidgets in a separate private attribute. (#2773)
+- Improve error message when legacy Qt installed from conda over pip (#2776)
+- Change remove to uninstall in plugin dialog (#2787)
+- Do not write settings loaded from environment values (#2797)
+
+
+## Bug Fixes
+- Fixing `add_dock_widget` compatibility with `magicgui v0.2` (#2734)
+- Shortcuts: Render properly shortcuts with minus and space. (#2735)
+- Fix runtime error when running doc tests on napari site (#2738)
+- Fix play button with dask/zarr (#2741)
+- Ignore opening 1D image layers in the viewer model. (#2743)
+- Fix custom layer subclasses (don't require layer icon) (#2758)
+- Add temporal fix for handling enums in qt_json_form (#2761)
+- Use slicing rather than np.take in add_image (#2780)
+- Fix paint cursor size for layer with negative scale (#2788)
+- track_id dtype change. from uint16 to uint32 (#2789)
+- Ensure aborted workers don't emit returned signal (#2796)
```

Comment:
In theory this would be an API change, although I very much doubt someone was using it this way...